Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -19,7 +19,18 @@ public extension Array {
         insert(newElement, at: 0)
     }
 
-    /// SwifterSwift: Safely swap values at given index positions.
+  /// SwifterSwift: Insert an element in the middle of the collection.
+  ///
+  ///       [1, 2, 3, 4].insertInMiddle(5) -> [1, 2, 5, 3 , 4]
+  ///       [1, 2, 3, 4, 5].insertInMiddle(6) -> [1, 2, 6, 3 , 4, 5]
+  ///
+  /// - Parameter newElement: element to insert.
+  mutating func insertInMiddle(_ newElement: Element) {
+      let middleIndex = index(startIndex, offsetBy: distance(from: startIndex, to: endIndex) / 2)
```

Comment:
```suggestion
      let middleIndex = index(startIndex, offsetBy: count/ 2)
```
Distance from `start` to `end` is the same as `count` :)